CMakeLists.txt

```diff
@@ -28,17 +28,23 @@ option(cga_device_synchronize_kernels "Run cudaDeviceSynchronize() in CGA_CU_CHE
 option(spoa_accurate "Run cudapoa code in mode that matches spoa" OFF)
 option(cga_enable_cudapoa_nw_print "Enable verbose prints within cudapoa NW kernel" OFF)
 option(cga_profiling "Compile a binary for profiling with NVTX markers." OFF)
+option(cga_generate_docs "Generate Doxygen documentation" ON)
 
+include(cmake/Utils.cmake)
```

Comment:
@ohadmo can you add a comment above this include saying that we have to include it before any of the other includes since it had utilities?